x-pack/plugins/maps/public/connected_components/edit_layer_panel/filter_editor/filter_editor.tsx

```diff
@@ -199,6 +200,7 @@ export class FilterEditor extends Component<Props, State> {
         onClick={this._toggle}
         data-test-subj="mapLayerPanelOpenFilterEditorButton"
         iconType={openButtonIcon}
+        disabled={this.props.editModeActiveForLayer}
```

Comment:
Can you update `_renderQuery` to display message "Layer filter is not applied while editing features" instead of showing filter when editing features

<img width="300" alt="Screen Shot 2022-03-28 at 8 22 24 AM" src="https://user-images.githubusercontent.com/373691/160419245-d37cfd35-c6e6-4b85-b467-e600748312d4.png">